src/control/mod.rs

```diff
@@ -445,8 +445,9 @@ pub trait Device: super::Device {
         let property = property::Info {
             handle,
             val_type,
-            mutable: info.flags & ffi::DRM_MODE_PROP_IMMUTABLE == 0,
-            atomic: info.flags & ffi::DRM_MODE_PROP_ATOMIC == 0,
+            mutable: !flags.contains(ModePropFlags::IMMUTABLE),
+            // TODO: this used to be the other way around!
+            atomic: flags.contains(ModePropFlags::ATOMIC),
```

Comment:
@Drakulix Forgot to leave a comment on this `TODO`... What's your thought? Fwiw, the kernel documents this property as follows:


```rust
/// the [`Self::ATOMIC`] flag is used to hide properties from userspace that
/// is not aware of atomic properties.  This is mostly to work around
/// older userspace (DDX drivers) that read/write each prop they find,
/// witout being aware that this could be triggering a lengthy modeset.
```

Reply:
Yeah, this was probably a copy/paste error from the line above. This seems correct. Please just remove the TODO

Reply:
Cool, should be done :)